### DISCUSSION TITLE
How to change background color for file list window or globablly as well as the search input?

### DISCUSSION THREAD

**Discussion by neonvoidx** [ibhagwan/fzf-lua#1580] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/63d8825f-1fb7-458d-9feb-71ce102d28d4)

Seems changing FzfLuaFzfNormal bg to any color other than none works, but not `none` which I would expect to make it transparent

Note: bg_float_configured is just a variable that changes based on transparent opt for my theme, either `none` or `c.bg_float`
Current highlight group setup with my theme:
```
    FzfLuaNormal = { fg = c.fg, bg = bg_float_configured },
    FzfLuaBorder = { fg = c.green, bg = bg_float_configured },
    FzfLuaBackdrop = { bg = bg_float_configured },
    FzfLuaPreviewNormal = { bg = bg_float_configured },
    FzfLuaPreviewBorder = { bg = bg_float_configured },
    FzfLuaHelpNormal = { bg = bg_float_configured },
    FzfLuaScrollBorderEmpty = { bg = bg_float_configured },
    FzfLuaScrollBorderFull = { bg = bg_float_configured },
    FzfLuaScrollFloatEmpty = { bg = bg_float_configured },
    FzfLuaScrolFloatFull = { bg = bg_float_configured },
    FzfLuaSearch = { fg = c.fg, bg = bg_float_configured },
    FzfLuaTitle = { fg = c.cyan, bg = bg_float_configured },
    FzfLuaFzfNormal = { bg = bg_float_configured, fg = c.fg },
    FzfLuaFzfCursorLine = { bg = bg_float_configured, fg = c.green },
    FzfLuaFzfPrompt = { bg = bg_float_configured, fg = c.green },
    FzfLuaFzfQuery = { bg = bg_float_configured, fg = c.cyan },
    FzfLuaDirPart = { bg = bg_float_configured, fg = c.comment },

```

**Comment by ibhagwan** [comment, marked as answer]
> Seems changing FzfLuaFzfNormal bg to any color other than none works, but not none which I would expect to make it transparent

That’s most likely because the highlight @none is cleared (I.e. no bg color) but for fzf colors, the bg color needs an actual value to be sent to fzf’s `--color` flag, for this (and also “gutter” there’s a better alternative, set the flag directly to `-1`:
```lua
:FzfLua files fzf_colors.bg=-1
```
If this is what you’re after add this to your setup (with the latest commit):
```lua
require('fzf-lua').setup({
  fzf_colors = {
    true, -- auto generate rest of fzf’s highlights?
    bg = '-1',
    gutter = '-1', -- I like this one too, try with and without
  }
})
```

**Reply by neonvoidx** [reply]
Yup thats exactly what I want, not going to lie, I get confused with whats the nvim plugin and whats fzf command stuff.

**Reply by ibhagwan** [reply]
> Yup thats exactly what I want, not going to lie, I get confused with whats the nvim plugin and whats fzf command stuff.

Understandable, if you haven’t used fzf in the shell extensively prior to using fzf-lua it takes a while to understand which parts are fzf-lua/neovim and what parts comes from the external fzf binary.
